docs/resources/secmaster_incident.md

```diff
@@ -0,0 +1,143 @@
+---
+subcategory: "SecMaster"
+---
+
+# huaweicloud_secmaster_incident
+
+Manages a SecMaster incident resource within HuaweiCloud.
+
```

Comment:
How ablout add a note description "This resource only support for "cn-east-3".

Reply:
sure